rxgroups-android/src/main/java/com/airbnb/rxgroups/LifecycleResubscription.java

```diff
@@ -63,7 +63,12 @@
 
     // Get all fields on the target object, including inherited fields
     while (shouldCheckClass(classToCheck)) {
-      list.addAll(Arrays.asList(classToCheck.getDeclaredFields()));
+      //noinspection OverlyBroadCatchBlock
+      try {
+        list.addAll(Arrays.asList(classToCheck.getDeclaredFields()));
+      } catch (Exception ignored) {
```

Comment:
cant catch `ClassNotFoundException` because it's a checked exception and it's not declared in `getDeclaredFields()`:
```java
public native Field[] getDeclaredFields();
```